ssh/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "che-theia-ssh-extension",
```

Comment:
@benoitf  @azatsarynnyy What package scope should we use for release? Or we don't want to use it? Theia for example use package scope "theia", so Theia npm packages for example calls with this package scope: "@theia/core", "@theia/workspace" and so on. And by this package scope we can quickly filter on the npm registry all packages for Theia.

Reply:
@AndrienkoAleksandr I have no idea what package scope should we use for our extensions.